pages/common/sdcv.md

```diff
@@ -0,0 +1,33 @@
+# sdcv
+
+> StarDict command-line version, dictionary client.
+> Dictionaries are provided separately from the client.
+> More information: <https://manned.org/sdcv>.
+
+- Start sdcv interactively:
+
+`sdcv`
+
+- List installed dictionaries:
+
+`sdcv --list-dicts`
+
+- Display a definition from a specific dictionary:
+
+`sdcv --use-dict {{dictionary_name}} {{search_term}}`
+
+- Lookup a definition with a fuzzy search:
```

Comment:
```suggestion
- Lookup a definition with fuzzy search:
```
Just apply if you want to

Reply:
Without the `a` this doesn't make grammatical sense?